packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -784,6 +785,7 @@ contract Superfluid is
     )
        internal
     {
+        bool valueForwarded = false;
```

Comment:
We should send all msg.value back if no app action is in the batch call that uses it.

Reply:
added